lepton/image.go

```diff
@@ -39,22 +39,6 @@ func BuildImage(c types.Config) error {
 	return nil
 }
 
-// rebuildImage rebuilds a unikernel image for user
-// supplied ELF binary after volume attach/detach
-func rebuildImage(c types.Config) error {
```

Comment:
looks like this and the refreshRate were removed because they aren't referenced which is fine but for future ref prob. best to include this in a sep. pr to eliminate any confusion